Makefile

```diff
@@ -259,8 +263,22 @@ generate-mocks: mockery ## Generate the mocks for the project as configured in .
 clean-ui:
 	find ${UI_DIR} -type f -name extension.tar -delete
 
+.PHONY: codegen-download-openapi run-backend
+
+codegen-download-openapi:
```

Comment:
please add `build` and `goreman` as a dependency of this target. Also add the documentation for the target. 
Use this as a reference:
https://github.com/argoproj-labs/argocd-ephemeral-access/blob/d2735f179f10db23ebc7a19269b5c8e02e69795d/Makefile#L106